### ISSUE TITLE
PWA: Camera does not work on iOS 12

### ISSUE THREAD

**Issue by matt-auckland** [Encrypt-S/NavPay#169]
Users on iOS 12 (currently the latest version) can not use the camera in the PWA.
Confirmed bug.

Replication:
- Open PWA on iOS.
- Open 'Scan'
- "Camera is not supported on this device" message is displayed

**Comment by ro-savage** [comment]
It is likely the code that checks if the camera is supported on iOS.

As I don't have an iOS device with me, someone who does will need to console.log this stuff out and see whats failing / what is being output and update the code accordingly.

Hopefully its just something simple.

https://github.com/Encrypt-S/NavPay/blob/8c72c939db94ad0edfb9619ce671622fafd42619/src/js/services/platformInfo.js#L67-L111

**Comment by ro-savage** [comment]
This line is incorrect:

https://github.com/Encrypt-S/NavPay/blob/8c72c939db94ad0edfb9619ce671622fafd42619/src/js/services/platformInfo.js#L71

should
```js
 ret.iOSPWASupport = ret.iOSVersion.length ? (ret.iOSVersion[0] >= 11 && ret.iOSVersion[1] >= 3) || (ret.iOSVersion[0] >= 12 ) : false 

```

**Comment by matt-auckland** [comment]
Fixed by #170
